agent/fw_wordpress.c

```diff
@@ -428,13 +379,21 @@ static char* nr_wordpress_clean_tag(const zval* tag TSRMLS_DC) {
     return NULL;
   }
```

Comment:
This check is not needed - `tag` is guaranteed not to be `NULL` [here](https://github.com/newrelic/newrelic-php-agent/blob/bd37e27f12817d771240a416caef01571a94670a/agent/fw_wordpress.c#L435) and [here](https://github.com/newrelic/newrelic-php-agent/blob/bd37e27f12817d771240a416caef01571a94670a/agent/fw_wordpress.c#L522).